scripts/initialize_local_chain.sh

```diff
@@ -37,13 +26,15 @@ jq '.validators[0] += {"pub_key":'$KEY'}' ~/.sei/config/tmp_genesis_2.json > ~/.
 mv ~/.sei/config/tmp_genesis_3.json ~/.sei/config/genesis.json && rm ~/.sei/config/tmp_genesis.json && rm ~/.sei/config/tmp_genesis_2.json
 
 echo "Creating Accounts"
-python3  loadtest/scripts/populate_genesis_accounts.py 50 loc
+# create 10 test accounts + fund them
+python3  loadtest/scripts/populate_genesis_accounts.py 10 loc
 
 ~/go/bin/seid collect-gentxs
+# update some params in genesis file for easier use of the chain localls (make gov props faster)
 cat ~/.sei/config/genesis.json | jq '.app_state["gov"]["deposit_params"]["max_deposit_period"]="300s"' > ~/.sei/config/tmp_genesis.json && mv ~/.sei/config/tmp_genesis.json ~/.sei/config/genesis.json
```

Comment:
Do we need this? Can we just use the default

Reply:
The default for gov is 2 days for normal and 1 day for expedited, which is too long for local chain